sdk/keyvault/azure-security-keyvault-jca/src/test/resources/logging.properties

```diff
@@ -0,0 +1,18 @@
+# To enable this configure file, Please add this property:
+# -Djava.util.logging.config.file="src/test/resources/logging.properties"
+#
+# The Java logging APIs (java.util.logging) default loads logging.properties from:
+# 1. $JAVA_HOME/jre/lib/ (Java 8 and before)
+# 2. $JAVA_HOME/conf/ (Java 9 and above)
+#
+# For more information about this file, please refer to:
+# 1. https://docs.oracle.com/javase/8/docs/technotes/guides/logging/overview.html#a1.8
+# 2. https://docs.oracle.com/cd/E23549_01/doc.1111/e14568/handler.htm
+
+handlers = java.util.logging.ConsoleHandler
+java.util.logging.ConsoleHandler.level = ALL
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+java.util.logging.SimpleFormatter.format= [%1$tF %1$tT] %3 [%4$-7s] %5$s %n
+
+.level = INFO
+com.azure.security.keyvault.jca.level = ALL
```

Comment:
What are these two for? I don't think it's fully explained in the Oracle documentation.

Reply:
Just set the log level for this: package `com.azure.security.keyvault.jca.level`.